nengo/dists.py

```diff
@@ -221,18 +221,32 @@ class UniformHypersphere(Distribution):
     surface : bool, optional (Default: False)
         Whether sample points should be distributed uniformly
         over the surface of the hyperphere (True),
-        or within the hypersphere (False).
+        or within the hypersphere (False). Defaults to False.
+    min_magnitude : Number, optional (Default: 0)
+        Lower bound on the returned vector magnitudes (i.e.
+        magnitude >= min_magnitude). Must be in the range [0, 1]; defaults
+        to 0. Values outside this range will be clipped to this range. Ignored
+        if ``surface == True``
     """
 
     surface = BoolParam('surface')
+    min_magnitude = NumberParam('min_magnitude', low=0, high=1)
```

Comment:
Add ``high_open=True`` so that we don't allow ``min_magnitude == 1``, which I think would cause problems because then we'd be sampling from an empty interval.